packages/kbn-pm/src/commands/watch_bazel.ts

```diff
@@ -6,23 +6,14 @@
  * Side Public License, v 1.
  */
 
-import { runIBazel } from '../utils/bazel';
+import { log } from '../utils/log';
 import { ICommand } from './';
 
 export const WatchBazelCommand: ICommand = {
-  description: 'Runs a build in the Bazel built packages and keeps watching them for changes',
+  description: 'Removed',
```

Comment:
Same here, I wouldn't be surprised if an email to contributors that says "run `yarn kbn watch` to watch all packages and `yarn kbn build` to build all packages" would be enough of a "deprecation" and we can just remove these commands.